experiment_design_doc.md

```diff
@@ -94,6 +94,16 @@ The experiment infrastructure consists of several components:
   performance of subjects using Tellina versus those who are not, via
   statistical analysis.  This will be done with a post-processing program.
 
+### User Requirements [Remote Edition]
+The subjects will participate to the experiment remotely. Thus, they will not be able to use CSE lab computers and will have to use their laptops.
```

Comment:
- "will participate ~to~ _in_ the experiment"
- "will have to use their _personal_ laptops"

Reply:
Thanks!